### DISCUSSION TITLE
Can you use lerna with yarn workspaces with cross-references?

### DISCUSSION THREAD

**Discussion by anlesk** [lerna/lerna#4104] [Q&A] [answered]
Newer versions of yarn does support [cross-references](https://yarnpkg.com/features/workspaces#cross-references), allowing to define the dependency between the components in the same monorepo as:
```{
  "dependencies": {
    "@my-org/utils": "workspace:*"
  }
}
```

Does lerna support that syntax? Would lerna version command work?
I tried that in the past before nx migration, and lerna version been failing to locate the version.

**Comment by literat** [comment, marked as answer]
Hi, I hope I can help with this.

We are using [cross-references in our monorepo](https://github.com/lmc-eu/spirit-design-system/blob/main/package.json#L91) managed by the Lerna.

It works just fine and we have this cross-reference in multiple packages.

This works because Lerna removed some legacy APIs like `add` and left all the heavy lifting up to the package manager. So this is now not a part of the Lerna but managed by the package manager itself.

I think that more can be found in the [documentation](https://lerna.js.org/docs/legacy-package-management) :-)

**Reply by anlesk** [reply]
This is amazing to hear! I'll give it a try at post back the results.
Thanks for the useful links too!

**Reply by anlesk** [reply]
Confirming it works and it is brilliant.
Thanks again, @literat!
